test/dev/integration.js

```diff
@@ -524,3 +524,28 @@ test('[now dev] no build matches warning', async t => {
     dev.kill('SIGTERM');
   }
 });
+
+test('[now dev] do not recursivly check the path', async t => {
+  const directory = fixture('trigger-static-build');
```

Comment:
Is this a copy+paste typo?

```suggestion
  const directory = fixture('handle-filesystem-missing');
```